src/Lumi/Components/Table.js

```diff
@@ -1,26 +1,26 @@
 "use strict";
 
 // This is here until we can get rid of Immutable
-exports.dataSize = function(data) {
+exports.dataSize = function (data) {
   return data.size ? data.size : data.length;
 };
 
-exports.getMouseEventPositionWithOffset = function(domNode, e) {
+exports.getMouseEventPositionWithOffset = function (domNode, e) {
   var bounds = domNode.getBoundingClientRect();
   return {
-    x: e.pageX - bounds.left,
-    y: e.pageY - bounds.top
+    x: e.pageX - window.scrollX - bounds.left,
```

Comment:
Was this broken in cases where the window was scrolled before?

Reply:
Yeah, I'm not sure when/why it changed, but scrolling to the bottom table example was enough to set the dropdown off the bottom of the screen. This places it back under the cursor when I tested it.